internal/auth/core/handler/auth_handler.go

```diff
@@ -125,41 +181,61 @@ func (ah *authHandler) LogInDogowner(c echo.Context, adoReq authDTO.AuthDogOwner
 			err,
 			"パスワードが間違っています",
 			wrErrors.NewAuthServerErrorEType())
-
 		logger.Errorf("Password compare failure: %v", wrErr)
-
-		return "", wrErr
+		return model.AuthDogOwner{}, wrErr
 	}
 
-	// 更新用のJWT IDの生成
-	jwtID, wrErr := GenerateJwtID(c)
+	return results[0].AuthDogOwner, nil
+}
 
-	if wrErr != nil {
-		return "", wrErr
-	}
+// VerifyRequestAndFetchAuthDogOwnerRefresh:
+//
+// args:
+//   - echo.Context:	コンテキスト
+//   - string:	リフレッシュトークン
+//
+// return:
+//   - model.AuthDogOwner:	認証ユーザー
+//   - error:	エラー
+func (ah *authHandler) VerifyRequestAndFetchAuthDogOwnerRefresh(c echo.Context, refreshToken string) (model.AuthDogOwner, error) {
+	logger := log.GetLogger(c).Sugar()
 
-	// 取得したdogownerのjtw_idの更新
-	if wrErr := ah.ar.UpdateDogownerJwtID(c, results[0].AuthDogOwner.DogOwner.DogOwnerID.Int64, jwtID); wrErr != nil {
-		return "", wrErr
+	claims := &AccountClaims{}
+
+	// 署名キーを指定
+	token, err := jwt.ParseWithClaims(refreshToken, claims, func(token *jwt.Token) (interface{}, error) {
```

Comment:
```suggestion
	token, err := jwt.ParseWithClaims(refreshToken, claims, func(token *jwt.Token) (any, error) {
```
interface{}じゃなくて、anyで統一してたわ
任せる